vendor/github.com/onsi/ginkgo/v2/Makefile

```
@@ -0,0 +1,16 @@
+.PHONY: all
+all:  vet test
+
+.PHONY: test
+test:
+	go run github.com/onsi/ginkgo/v2/ginkgo -r -p -randomize-all -keep-going
+
+.PHONY: vet
+vet:
+	go vet ./...
+
+.PHONY: update-deps
```

checkmake (0.2.2)

[warning] Missing required phony target "clean"

(minphony)